compression/zinfo.h

```diff
@@ -53,9 +53,29 @@
 typedef unsigned char uchar;
 typedef int64_t offset_t;
 
+#define ZINFO_VERSION_ONE 1
+#define ZINFO_VERSION_TWO 2
+
+#define ZINFO_VERSION_CUR ZINFO_VERSION_TWO
+
 /* Since gzip is compressed with 32 KiB window size, WINDOW_SIZE is fixed */
 #define WINSIZE 32768U
 
+/*
+    -  8 bytes, compressed offset
+    -  8 bytes, uncompressed offset
+    -  1 byte, bits
+    -  32768 bytes, window
+*/
+#define PACKED_CHECKPOINT_SIZE (8 + 8 + 1 + WINSIZE)
+
+/*
+    -  4 bytes, number of checkpoints
+    -  8 bytes, span size
+*/
+#define BLOB_FIXED_SIZE (4 + 8)
+
+
```

Comment:
nit: `BLOB_FIXED_SIZE` -> `FIXED_BLOB_SIZE` (more consistent with `PACKED_CHECKPOINT_SIZE`)

Reply:
I was trying to use `fixed size` as a noun phrase here to refer to the "fixed size" portion of the blob. I think `FIXED_BLOB_SIZE` makes it sound like `blob size` is a noun phrase, but that means something different. 

To be fair, it would probably be easier to read if both of these 3-word constants placed the noun phrase in the same place, so what about `PACKED_CHECKPOINT_SIZE` and `BLOB_HEADER_SIZE`?

Reply:
SGTM, that's clear. After all `PACKED_CHECKPOINT_SIZE` is also a fixed size by itself based on the calculation in it's comment. :)